config/app.php

```diff
@@ -336,10 +336,10 @@
              */
             // 'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
 
+            'host' => env('DATABASE_HOST'),
```

Comment:
Let's try to fetch `DATABASE_SOCKET` in case `DATABASE_HOST` is not there. Then we can merge, which would deploy, things will still work and we can adjust the Terraform definitions as a secondary step.

Reply:
959ab319daff74cd6391b26b11b40348b966e5fb